client/src/components/RatingCriterion/RatingCriterion.css

```diff
@@ -0,0 +1,9 @@
+.crit-name {
+  font-size: large;
+  margin: 5px 0px 0px 0px;
```

Comment:
```
2021-08-05 06:15:48 [ERROR]   Error code: 2. Command output:
90
------
91

92
client/src/components/RatingCriterion/RatingCriterion.css
93
 3:16  ✖  Unexpected unit   length-zero-no-unit
94
 3:20  ✖  Unexpected unit   length-zero-no-unit
95
 3:24  ✖  Unexpected unit   length-zero-no-unit
96
------
```

The unit issue is talking about here. It’s saying thst 0 doesn’t need a unit. So just change the `0px` to `0`.